clients/iceberg-views/src/main/java/org/apache/iceberg/nessie/NessieViewOperations.java

```diff
@@ -26,7 +29,11 @@
 import org.apache.iceberg.io.FileIO;
 import org.apache.iceberg.util.Tasks;
 import org.apache.iceberg.view.BaseMetastoreViewOperations;
+import org.apache.iceberg.view.BaseVersion;
+import org.apache.iceberg.view.HistoryEntry;
+import org.apache.iceberg.view.Version;
```

Comment:
Does this mean we have a circular dependency with Iceberg?

Reply:
yes but the code here will move eventually into iceberg